modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarEvent.macro

```diff
@@ -746,9 +746,11 @@ definition {
 	}
 
 	macro disableEventBySiteCalendarName {
-		Click(
-			key_calendarName = ${calendarName},
-			locator1 = "Calendar#CURRENT_SITE_CALENDARS_CALENDAR_PRIMARY");
+		if (IsElementPresent(locator1 = "Calendar#CURRENT_SITE_CALENDARS_CALENDAR_PRIMARY_COLOR_DEFAULT")) {
+			Click(
+				key_calendarName = ${calendarName},
+				locator1 = "Calendar#CURRENT_SITE_CALENDARS_CALENDAR_PRIMARY");
+		}
```

Comment:
Hey @thaiscabrals, I don't see the need for an if statement in a macro. If the locator is not present, then the macro will not do anything.

Can you take a look at this [commit](https://github.com/liferay-bpm-qa/liferay-portal/pull/831/commits/639a091566e3872c47909f8d7c6aeb38812c7903) and see if doing that will fix the issue? It looks like the original issue is not being able to find the Liferay DXP site and the reason for that is because our tests on CI run on virtual instances

Reply:
I thought it would be more specific. But you are right. thanks for that!
I have already removed the `if` from the macro.